packages/viewer/src/index.html

```diff
@@ -28,7 +28,8 @@
     </style>
 
     <script type="module">
-      window.addEventListener('DOMContentLoaded', () => {
+      window.addEventListener('DOMContentLoaded', async () => {
+        await window.customElements.whenDefined('vertex-viewer');
```

Comment:
I think it'd better communicate that we need to wait for the component to be hydrated if this was moved to `main`, since we're querying for the element there.

Reply:
That sounds good. If we go with this route, I will move it.

Going to look into using `defineCustomElements` instead